readthedocs/builds/forms.py

```diff
@@ -29,9 +33,22 @@ def __init__(self, instance=None, *args, **kwargs):  # noqa
 
 class VersionForm(forms.ModelForm):
 
+    slug = forms.CharField(
+        max_length=255,
+        validators=[RegexValidator('^{pattern}$'.format(pattern=VERSION_SLUG_REGEX))],
+        help_text=_("Used in this version's URL"),
+    )
+
     class Meta(object):
         model = Version
-        fields = ['active', 'privacy_level', 'tags']
+        fields = ['slug', 'active', 'privacy_level', 'tags']
+
+    def __init__(self, *args, **kwargs):
+        super(VersionForm, self).__init__(*args, **kwargs)
+        if self.instance and self.instance.slug in (LATEST, STABLE):
+            self.fields['slug'].disabled = True
```

Comment:
Besides making the field gray in the UI, does this validates this field in the backend also?